action.yml

```diff
@@ -14,18 +14,24 @@ inputs:
   install-dir:
     description: 'Where to install the cosign binary'
     required: false
-    default: '$HOME/.cosign'
```

Comment:
this might have a breaking impact on folks if they're hardcoded the path, perhaps we should include a comment in the changelog about this